lib/rubocop/cop/style/hash_syntax.rb

```diff
@@ -76,6 +76,9 @@ module Style
       #   # good
       #   {foo:, bar:}
       #
+      #   # good - allowed to mix syntaxes
+      #   {foo:, bar: baz}
```

Comment:
For `bar: baz` in `{ foo:, bar: baz }`, it's obvious that the value can't be omitted, so including it in the documentation would be redundant. While "allowed to mix syntaxes" likely refers to the style `{ foo:, bar: bar }`, that form isn’t allowed in this style.

Reply:
> This is in contrast to `EnforcedShorthandSyntax: consistent` where you're *not* allowed to mix syntaxes.

Ah, I see. This makes for symmetry in the documentation. It would’ve been helpful to have that noted in the PR description :-)

Reply:
Yeah, in retrospect it would have :) Cheers!